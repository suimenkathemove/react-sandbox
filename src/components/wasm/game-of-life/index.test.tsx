import { cleanup, render } from '@testing-library/react';

import { defaultProps } from './index.stories';

import { GameOfLife } from './';

const setup = () => render(<GameOfLife {...defaultProps} />);

xdescribe('GameOfLife', () => {
  afterEach(() => {
    cleanup();
  });

  test('render', () => {
    const { asFragment } = setup();
    expect(asFragment()).toMatchSnapshot();
  });
});
