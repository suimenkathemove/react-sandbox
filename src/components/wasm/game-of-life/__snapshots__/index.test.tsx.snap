// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GameOfLife render 1`] = `
<DocumentFragment>
  <div />
</DocumentFragment>
`;
